lib/packages_manager.py

```diff
@@ -50,7 +47,7 @@ def prepare_packages(self, packages_class=Package,
             self.packages.append(package)
 
 
-def discover_packages():
+def discover_packages(versions_repo_path):
```

Comment:
Isn't it simpler if this modules just calls `versions_repository.get_versions_repo`?